product_docs/docs/biganimal/release/overview/03a_region_support.mdx

```diff
@@ -2,9 +2,11 @@
 title: "Supported regions"
 ---
 
-## Supported Azure regions
+## Supported Azure regions 
 
-When using Azure, you can create a cluster in the following regions:
+When using Azure, you can create clusters in the following regions:
+
+**Single node and high availability clusters**
```

Comment:
Can we make this a table instead of two lists?

Reply:
Not easily. Markdown doesn't support lists in tables. I could play around with it using some HTML if you feel strongly about it being a preferred format.